### PR TITLE
chore: restyle listings search modal

### PR DESCRIPTION
**Pull request by isaac-looker**
# Pull Request Template

## Issue Overview

This PR addresses restyle search modal work

dw-final-modal

Figma

figma

- [ ] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Please include a summary of the change and which issue(s) is addressed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## How Can This Be Tested/Reviewed?

Provide instructions so we can review.

Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: sites/public/src/components/listings/search/ListingsSearchModal.tsx [hunk 1]
```diff
@@ -146,66 +176,82 @@ export function ListingsSearchModal(props: ListingsSearchModalProps) {
...
       open={props.open}
       onClose={props.onClose}
-      title={"Filters"}
+      title="Filters"
+      headerClassNames="bottom-border"
       ariaDescription="Listing Search Filters"
       actions={[
-        <button onClick={onSubmit}>Show matching listings</button>,
+        <Button type="button" className="is-secondary" onClick={onSubmit}>
+          Show matching listings
```
Comment (ifranch) [inline sites/public/src/components/listings/search/ListingsSearchModal.tsx:184]: All of the text in the buttons and labels should be translated, can you add it to general.json and call t() or do it as a follow up? My PR has it if you want to merge that but there are some extra labels in your PR

File: sites/public/src/components/listings/search/ListingsSearchModal.tsx [hunk 2]
```diff
@@ -1,17 +1,47 @@
...
+  Field,
+  FieldGroup,
+  FieldSingle,
+} from "@bloom-housing/doorway-ui-components"
 import { useForm } from "react-hook-form"
 
 const inputSectionStyle: React.CSSProperties = {
   margin: "0px 15px",
-  borderTop: "1px solid black",
+  // borderTop: "1px solid var(--bloom-color-gray-500)",
```
Comment (phattya) [inline sites/public/src/components/listings/search/ListingsSearchModal.tsx:16]: should this be deleted?

File: sites/public/src/components/listings/search/ListingsSearchModal.tsx [hunk 3]
```diff
@@ -1,17 +1,47 @@
...
+const hyphenContainerStyle: React.CSSProperties = {
+  position: "relative",
+}
+
+const hyphenStyle: React.CSSProperties = {
+  fontSize: "2rem",
+  position: "relative",
+  bottom: "1px",
+  padding: ".4rem",
+  // offsetPosition: "top 3rem",
```
Comment (phattya) [inline sites/public/src/components/listings/search/ListingsSearchModal.tsx:28]: should this be deleted?

File: sites/public/src/components/listings/search/ListingsSearchModal.tsx (hunk 1 above)
Comment (phattya) [reply sites/public/src/components/listings/search/ListingsSearchModal.tsx:184]: Agreed. If you don't get to it today, I can get to it next week w/the translations, if needed.

**Comment by phattya** [comment]
Can we make the checks in the checkboxes black, like the mocks?  If it's not a simple change, feel free to push to next week.
image

Mocks: 
image

**Review by phattya — APPROVED** [review]
Isaac, wonderful job!!! This looks great & all the testing I did worked perfectly!

Just a few styling changes to make.  Is it possible to add the map today too?  If not, maybe first thing on Tuesday?

File: doorway-ui-components/src/forms/Field.tsx [hunk 4]
```diff
@@ -129,7 +132,12 @@ const Field = (props: FieldProps) => {
   if (props.note) {
     note = <p className="field-note mb-4">{props.note}</p>
   }
-
+  const inputClasses: string[] = ["input"]
+  if (props.inputClassName) {
+    // Move the input class(es) ahead of the default.
+    inputClasses.unshift(props.inputClassName)
+  }
+  console.log("inputClasses", inputClasses)
```
Comment (chriscasto) [inline doorway-ui-components/src/forms/Field.tsx:140]: This should be removed

File: doorway-ui-components/src/global/forms.scss [hunk 5]
```diff
@@ -70,8 +70,9 @@
     .input {
       @apply border;
       @apply border-gray-500;
-      @apply bg-gray-200;
-      @apply rounded;
+      // @apply bg-gray-200;
```
Comment (chriscasto) [inline doorway-ui-components/src/global/forms.scss:73]: These should probably be removed, too.

File: sites/public/src/components/listings/search/ListingsSearchModal.tsx (hunk 1 above)
Comment (isaac-looker) [reply sites/public/src/components/listings/search/ListingsSearchModal.tsx:184]: These translations will go into `doorway-ui-components/src/locales/general.json` or another general.json file?

**Review by chriscasto — APPROVED** [review]
Looks good, just a few non-blocking nits.

The hyphen between the min and max rent isn't centered vertically, making it look a bit weird.  Probably not worth spending a ton of cycles on, but getting that lined up better would improve the overall look, IMO.
image

File: sites/public/src/components/listings/search/ListingsSearchCombined.tsx [hunk 6]
```diff
@@ -230,6 +224,12 @@ const locations: FormOption[] = [
     label: "Sonoma",
     value: "Sonoma",
   },
+  {
+    label: "San Francisco",
+    value: "San Francisco",
+    isDisabled: true,
+    labelNoteHTML: `(For San Francisco listings, please go to <a href="https://housing.sfgov.org/">DAHLIA</a>)`,
```
Comment (phattya) [inline sites/public/src/components/listings/search/ListingsSearchCombined.tsx:231]: Can we have the target=_blank here?  Thanks!

File: sites/public/src/components/listings/search/ListingsSearchModal.tsx (hunk 1 above)
Comment (isaac-looker) [reply sites/public/src/components/listings/search/ListingsSearchModal.tsx:182]: I'll resolve translations next week.

**Pull request merged**